Jenkinsfile

```diff
@@ -282,6 +282,21 @@ pipeline {
         githubNotify(context: "${env.GITHUB_CHECK_ITS_NAME}", description: "${env.GITHUB_CHECK_ITS_NAME} ...", status: 'PENDING', targetUrl: "${env.JENKINS_URL}search/?q=${env.ITS_PIPELINE.replaceAll('/','+')}")
       }
     }
+    stage('Stable') {
+      options { skipDefaultCheckout() }
+      when {
+        branch 'master'
```

Comment:
Let's remark this, just to test that the tag gets created.

Reply:
Do you mean to change the stage name?

```
stage('Create stable tag') {
```

Reply:
Otherwise, if something bad happened, the build will fail

Reply:
No, I meant to allow this to execute on any branch (like this one) to see that the tag is created and replaced properly after a commit

Reply:
Gotcha, though, I'll add some changes and revert them afterwards

Reply:
https://github.com/elastic/apm-agent-java/pull/1747/commits/afc5b7a9816d7c42f56cd16819b98c44beca118f is the commit with the changes:
- run immediately after the checkout
- run even on a PR basis.

Reply:
Build [#3](https://apm-ci.elastic.co/job/apm-agent-java/job/apm-agent-java-mbp/job/PR-1747/3/) created the tag:
- https://github.com/elastic/apm-agent-java/tree/stable

![image](https://user-images.githubusercontent.com/2871786/113843939-eb5e3480-978b-11eb-9ab5-97e6f5f3567f.png)

Build [#4](https://apm-ci.elastic.co/job/apm-agent-java/job/apm-agent-java-mbp/job/PR-1747/4/) forced a retag

![image](https://user-images.githubusercontent.com/2871786/113844467-6b849a00-978c-11eb-8887-81969bca3743.png)


**NOTE**: bear in mind, the commit history might differ between branches and PRs, since there is no a rebase on branches but on PRs.

Reply:
OK, let's restore and see how it works. Thanks!